doc/tutorials/pre-launched-rt.rst

```diff
@@ -44,15 +44,16 @@ kernels are loaded as multiboot modules. The ACRN hypervisor, Service
 VM, and Pre-Launched RT kernel images are all located on the NVMe drive.
 We recommend installing Ubuntu on the NVMe drive as the Service VM OS,
 which also has the required GRUB image to launch Pre-Launched RT mode.
-Refer to :ref:`Run Ubuntu as the Service VM <Ubuntu Service OS>`, to
+Refer to :ref:`rt_industry_ubuntu_setup`, to
 install Ubuntu on the NVMe drive, and use grub to launch the Service VM.
 
 Install Pre-Launched RT Filesystem on SATA and Kernel Image on NVMe
 ===================================================================
 
-The Pre-Launched Preempt RT Linux use Clearlinux as rootfs. Refer to
-:ref:`Burn the Preempt-RT VM image onto the SATA disk <install_rtvm>` to
-download the RTVM image and burn it to the SATA drive. The Kernel should
+.. important:: Need to add instructions to download the RTVM image and burn it to the
```

Comment:
@fuzhongl I left "important" notes in a few documents to prompt you for additional material needed when you make your updates (in another PR).

Reply:
@dbkinder Thank you!
Will follow it in my PR.

Reply:
@fuzhongl OK.  I'll talk with @NanlinXie and recommend we merge my PR (if there are no other issues) and then you can follow with PRs for the docs you're working on.